packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -800,6 +806,655 @@ let rows = [
+            {[...headerRow.childNodes].map(column => (
+              <ResizableTableColumnHeader
+                key={column.key}
+                column={column}
+                state={state}
+                /*- begin highlight -*/
+                layoutState={layoutState}
+                onResizeStart={onResizeStart}
+                onResize={onResize}
+                onResizeEnd={onResizeEnd}
+                /*- end highlight -*/
+              />
+            ))}
+          </ResizableTableHeaderRow>
+        ))}
+      </ResizableTableRowGroup>
+      <ResizableTableRowGroup
+        /*- begin highlight -*/
+        style={{
+          maxHeight: '200px'
+        }}
+        /*- end highlight -*/
+        type="tbody">
+        {[...collection.body.childNodes].map(row => (
+          <ResizableTableRow key={row.key} item={row} state={state}>
+            {[...row.childNodes].map(cell => (
+              <ResizableTableCell
+                key={cell.key}
+                cell={cell}
+                state={state}
+                /*- begin highlight -*/
+                widths={widths}
+                /*- end highlight -*/
+              />
+            ))}
+          </ResizableTableRow>
+        ))}
+      </ResizableTableRowGroup>
+    </table>
+  );
+}
+```
+
+### Resizable table header
+
+The `TableRowGroup` and `TableHeaderRow` only require minor style changes to accommodate the new `display` changes made in
+`ResizableColumnsTable`.
+
+```tsx example export=true render=false
+function ResizableTableRowGroup({type: Element, style, children}) {
+  let {rowGroupProps} = useTableRowGroup();
+  return (
+    <Element
+      {...rowGroupProps}
+      style={{
+        /*- begin highlight -*/
+        display: 'block',
+        /*- end highlight -*/
+        ...style
+      }}>
+      {children}
+    </Element>
+  );
+}
+```
+
+```tsx example export=true render=false
+function ResizableTableHeaderRow({item, state, children}) {
+  let ref = useRef();
+  let {rowProps} = useTableHeaderRow({node: item}, state, ref);
+
+  return (
+    <tr
+      {...rowProps}
+      ref={ref}
+      /*- begin highlight -*/
+      style={{display: 'flex'}}
+      /*- end highlight -*/
+    >
+      {children}
+    </tr>
+  );
+}
+```
+
+The `TableColumnHeader` is where we see the bulk of the changes required to support resizable columns. First of all, we need to accommodate a `Resizer` element in every resizable column that
+the user can drag or focus to perform a resize operation. To avoid interfering with the existing grid keyboard navigation, we only display this resizer when
+the column is hovered or if the column itself is being resized. This however introduces another issue: how will keyboard, screen reader, and touch users begin a resize operation if the resizer
+is only visible on hover? We can't just have the user press on the column header since that action is reserved for sorting.
```

Comment:
qq, we no longer support sorting here, can we actually do away with the menu entirely and just have clicking on the column header go into resize mode for this example?

the tailwind one supports sorting and shows how they can work together.

Reply:
I considered doing that but it felt rather non-standard and a bit divergent from the tailwind example. I'd have to add aria-labeling/message to let screenreader users know that pressing on the header would trigger resizing and demonstrate overriding handlers from `useTableColumnHeader` in the off chance sorting was enabled as well. 

Personally I like having the menu here since it serves as a nice step 1.5 for people to consider what to do if they wanted additional actions on the TableColumnHeader. Happy to discuss if we still feel that the menu is over complicating things here

Reply:
I'm fine with it as is.
We should update the sentence this is commenting on since the column header in this example is not reserved for sorting though.